gems/pending/Gemfile

```diff
@@ -50,7 +50,7 @@ gem "trollop",                 "~>2.0",             :require => false
 gem "uuidtools",               "~>2.1.3",           :require => false
 gem "winrm",                   "~>1.7.2",           :require => false
 gem "winrm-elevated",          "~>0.4.0",           :require => false
-gem 'sys-proctable',           "~> 1.0",            :require => false
+gem 'sys-proctable',           "~> 1.1",            :require => false
```

Comment:
I think this should be `"~> 1.1.0"`, because bundler takes the last digit in that string and will auto version it, so this means this could be anything between `1.1.X` to `1.[INFINITY].X`, instead of `1.1.0` to `1.1.X`.

This is actually how I noticed you had updated the gem because I ran a `bin/update` around 5pm after the gem was released and it had already updated.  We might want to audit our Gemfiles and make sure that we aren't doing this anywhere else and auto upgrading gems when we shouldn't and introducing instabilities.